client/Lykke.Service.PayInternal.Client/Lykke.Service.PayInternal.Client.csproj

```diff
@@ -24,4 +24,9 @@
     <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.10" />
     <PackageReference Include="Refit" Version="4.1.0" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.ComponentModel.Annotations">
+      <HintPath>C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.netcore.app\2.0.0\ref\netcoreapp2.0\System.ComponentModel.Annotations.dll</HintPath>
```

Comment:
Do we need to include absolute paths in csproj?

Reply:
It just a hint.